WalletWasabi/Tor/TorProcessManager.cs

```diff
@@ -259,8 +271,10 @@ public async Task StopAsync(bool killTor = false)
 				}
 			}
 
+			// Dispose Tor process resources (does not stop/kill Tor process).
 			TorProcess?.Dispose();
-			TorProcess = null;
```

Comment:
Why was this removed?

Reply:
In short: It's not necessary. 

* People used to make variables null a lot because it used to helped .NET's garbage collector. 
* It also may make sense to null a variable when you re-use it.

So I don't see a reason to have this assignment.

Reply:
That's not the reason why we make variables null. We do it, so if someone wants to access it with the `?` operators it'll just not access it. If you don't make it null, then when you want to access it even with the `?` operator, then it'll produce some very strange results as it was already disposed.

Reply:
Regardless if you're sure it's not necessary here, then I'm ok with that. We have better patterns already than to encounter many problems like this.

Reply:
Right. But then it may hide a bug. Weird behavior at least shows that the program is not correct. Anyhow, in this case, it is not necessary.